Templates/DefaultProject/Template/.command_settings

```diff
@@ -0,0 +1,6 @@
+[export_project]
+project.build.config = release
+archive.output.format = zip
+seedlist.paths = AssetBundling/SeedLists/*.seed
+option.build.assets = True
+option.build.tools = True
```

Comment:
For better support with installer builds, I think we need to add a default value for the monolithic builds:
```
option.build.monolithic= True
```

Reply:
alternative is to provide a `-mono` flag in the `export.bat` script

Reply:
That is default to true globally already, so no need to set per project.
https://github.com/o3de/o3de/blob/f0c150d75722b3302753972b28d73a8036b70b31/scripts/o3de/o3de/export_project.py#L1141-L1144